### PR TITLE
feat(mojaloop/project#2572): add terraform automation to docs infrastructure

### PR DESCRIPTION
**Pull request by lewisdaly**
- deploys updated terraform config upon merge to `mojaloop-docs-2.0` branch
- Allows us to specify redirects (see `./infra/src/redirect/index.js`)
- fixes some files ending with `.PNG`
- adds remote terraform state management

### CODE REVIEW

**Comment by lewisdaly** [comment]
Note: the build is failing because... well the vuepress build is failing! We didn't have a CI/CD check on this earlier, and I'm not going to fix the build in this PR to keep it as small as possible.

**Comment by lewisdaly** [comment]
@meekg33k please review when you get the chance - the sooner we merge, the sooner we can get to fixing some of the other ci/cd stuff

File: infra/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# infra
```
Comment (elnyry-sam-k) [inline infra/README.md:1]: hi @lewisdaly - this is directed towards administrators of Mojaloop docs, isn't it? or is it more general?

**Review by elnyry-sam-k — APPROVED** [review]
just one minor comment, otherwise looks good.. can't validate the terraform scripts but supposing the results will address that :-)

File: infra/README.md (hunk 1 above)
Comment (elnyry-sam-k) [reply infra/README.md:1]: do you think it would help add a note / statement to that effect?

**Review by elnyry-sam-k — APPROVED** [review]
Thanks!

**Review by meekg33k — APPROVED** [review]

**Pull request merged**